### DISCUSSION TITLE
Github student authentication fails due to an error in Github. What should I do?

### DISCUSSION THREAD

**Discussion by suku1515** [community/community#58899] [GitHub Education]
### Hi everyone,

I am unable to attempt student authentication right now because of an error in Github. The reason is that the name of the school was listed incorrectly. I tried to change the name of this school because it was not listed correctly, so I made it impossible to change it. So, no matter how much I tried with my student ID and certificate of enrollment, I couldn't do it because of the same "school name is not listed". I tried to look for this problem, but I couldn't find that the name of the school was different in ah, so I couldn't try it. What should I do? help!

**Comment by samus-aran** [comment]
Hi @suku1515 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the [same comment in multiple places](https://github.com/orgs/community/discussions/58825). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
